Project 1/templates/mainpage.html

```diff
@@ -1,12 +1,42 @@
 
 
 
+<script src="{{url_for('static',filename = 'script.js')}}" ></script>
 
 {% extends "layout.html" %}
-
 {% block banner %}
 	Welcome {{name}}
 {%endblock%}
 
+{% block body %}
+<form class="form-inline" >
+  <div class="form-group mx-sm-3 mb-2">
+    <label for="inputPassword2" class="sr-only">isbn</label>
+    <input type="text" name='isbn' class="form-control"  placeholder="isbn">
+  </div>
+  <div class="form-group mx-sm-3 mb-2">
+    <label for="inputPassword2" class="sr-only">title</label>
+    <input type="text" name='title' class="form-control"  placeholder="title">
+  </div>
+  <div class="form-group mx-sm-3 mb-2">
+    <label for="inputPassword2" class="sr-only">author</label>
+    <input type="text" name='author' class="form-control"  placeholder="author">
+  </div>
+  <div class="form-group mx-sm-3 mb-2">
+    <label for="inputPassword2" class="sr-only">year</label>
+    <input type="text" name='year' class="form-control" placeholder="year">
+  </div>
+  <button class="btn btn-primary mb-2">Search</button>
+</form>
+
```

Comment:
@kamujusrujan Where did you bind the addrev function written in script.js in this html page? Or addrev is being called from bookdetails page?

Reply:
it is being called from book details page.

Reply:
What I want to know is your addrev function must be called when some one clicks on submit the review button? So, I want to knoe where is that binding?

Reply:
there is an another function named review_book. this is the function that's called when we click on submit review button.

Reply:
@kamujusrujan Okie